cx_Freeze/freezer.py

```diff
@@ -279,6 +279,18 @@ def _freeze_executable(self, exe: Executable) -> None:
             copy_dependent_files=False,
             include_mode=True,
         )
+
+        # copy a file with a the cx_freeze license into frozen application
+        respath = get_resource_file_path(
+            "freeze_license", "cx_frozen_license", ".txt"
+        )
```

Comment:
../doc/src

Reply:
Maybe you can put this txt file in bases or initscripts.

Reply:
I'll put it in initscripts.